modules/mono/csharp_script.cpp

```diff
@@ -2335,17 +2342,18 @@ void CSharpScript::_placeholder_erased(PlaceHolderScriptInstance *p_placeholder)
 
 #ifdef TOOLS_ENABLED
 void CSharpScript::_update_exports_values(HashMap<StringName, Variant> &values, List<PropertyInfo> &propnames) {
-	if (base_cache.is_valid()) {
-		base_cache->_update_exports_values(values, propnames);
-	}
-
 	for (const KeyValue<StringName, Variant> &E : exported_members_defval_cache) {
 		values[E.key] = E.value;
 	}
 
+	propnames.push_back(get_class_category());
```

Comment:
In GDScript you are missing this line, not sure if this was an oversight.

Reply:
In gdscript this line is added on another place. I need to see how to do a mono build to see whether c# works

Reply:
To be more precise GDScript usss placholder script when the script is not defined as @tool